code/API_definitions/NetworkSliceBooking.yaml

```
@@ -0,0 +1,430 @@
+openapi: 3.0.3
+info:
+  title: API for Network Slice Booking
+  description: |
+    The Network Slice Booking (NSB) API provides programmable interface for developers to reserve a slice resource of a selected area within a period, and manage device access control as needed.
+  version: wip
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+servers:
+  - url: "{apiRoot}/nsb/vwip"
+    variables:
+      apiRoot:
+        default: http://localhost:9100

```

As in https://github.com/camaraproject/Commonalities/blob/main/documentation/API-design-guidelines.md#servers-object:
 
```suggestion
        default: http://localhost:9100
        description: API root, defined by the service provider, e.g. `api.example.com` or `api.example.com/somepath`
```